scale/product/serializers.py

```diff
@@ -13,20 +15,28 @@ class ProductFileBaseSerializer(ScaleFileBaseSerializer):
     is_operational = serializers.BooleanField()
     is_published = serializers.BooleanField()
     has_been_published = serializers.BooleanField()
+    is_superseded = serializers.BooleanField()
 
     published = serializers.DateTimeField()
     unpublished = serializers.DateTimeField()
+    superseded = serializers.DateTimeField()
 
     job_type = ModelIdSerializer()
     job = ModelIdSerializer()
     job_exe = ModelIdSerializer()
 
+    recipe_type = ModelIdSerializer()
+    recipe = ModelIdSerializer()
+    batch = ModelIdSerializer()
+
```

Comment:
Add source_started and source_ended here as well, don't forget to update the REST API docs too.